test/base/MockConnectors.scala

```diff
@@ -103,4 +98,7 @@ trait MockConnectors extends MockitoSugar {
     when(mockNrsConnector.submitNonRepudiation(any())(any(), any()))
       .thenReturn(Future.successful(NrsSubmissionResponse("submissionId1")))
 
+  def successfullyCancelDeclarationResponse(response: Boolean) =
```

Comment:
I'd rather slightly change the name to 'successfulCancelDeclarationResponse'.
Current name suggests different behaviour.